### PR TITLE
Bump Logging etc.

### PR DESCRIPTION
**Pull request by alexander-yevsyukov**
This PR updates the dependency on Spine Logging to latest version, addressing the breaking API changes.

External dependencies were also udpated.

### CODE REVIEW

File: src/main/java/io/spine/code/proto/Linker.java [hunk 1]
```diff
@@ -30,27 +30,31 @@
...
 import static java.util.stream.Collectors.toList;
+import static kotlin.jvm.JvmClassMappingKt.getKotlinClass;
 
 /**
  * Builds a set of {@link FileDescriptor}s from a list of {@link FileDescriptorProto}.
  */
 final class Linker {
 
-    private static final FluentLogger logger = FluentLogger.forEnclosingClass();
+    private static final Logger<?> logger = LoggingFactory.getLogger(getKotlinClass(Linker.class));
```
Comment (armiol) [inline src/main/java/io/spine/code/proto/Linker.java:56]: Let's have a comment on this Kotlin-specific lookup used. It's not too obvious.

File: src/main/java/io/spine/io/Delete.java [hunk 2]
```diff
@@ -26,20 +26,23 @@
...
+import static java.lang.String.format;
+import static kotlin.jvm.JvmClassMappingKt.getKotlinClass;
 
 /**
  * Utilities for delete operations on a file system.
  */
 public final class Delete {
 
-    private static final FluentLogger logger = FluentLogger.forEnclosingClass();
+    private static final Logger<?> logger = LoggingFactory.getLogger(getKotlinClass(Delete.class));
```
Comment (armiol) [inline src/main/java/io/spine/io/Delete.java:45]: Same as above.

File: src/main/java/io/spine/type/KnownTypes.java [hunk 3]
```diff
@@ -308,7 +311,8 @@ private List<String> allUrlList() {
     @Internal
     public static final class Holder {
 
-        private static final FluentLogger logger = FluentLogger.forEnclosingClass();
+        private static final Logger<?> logger =
+                LoggingFactory.getLogger(getKotlinClass(Holder.class));
```
Comment (armiol) [inline src/main/java/io/spine/type/KnownTypes.java:315]: And here.

**Review by armiol — APPROVED** [review]
@alexander-yevsyukov please consider my comments. Otherwise, LGTM.

**Comment by alexander-yevsyukov** [comment]
@dmdashenkov, FYI.

**Pull request merged**